titus-common/src/main/java/com/netflix/titus/common/util/DateTimeExt.java

```diff
@@ -58,6 +59,13 @@ public static String toUtcDateTimeString(long msSinceEpoch) {
         return ISO_UTC_DATE_TIME_FORMATTER.format(Instant.ofEpochMilli(msSinceEpoch)) + 'Z';
     }
 
+    public static String toLocalDateTimeString(long msSinceEpoch) {
+        if (msSinceEpoch == 0L) {
+            return null;
+        }
+        return ISO_LOCAL_DATE_TIME_FORMATTER.format(Instant.ofEpochMilli(msSinceEpoch)) + 'Z';
```

Comment:
I think the `Z` is reserved for times in UTC, we shouldn't append it in this case